epsagon/runners/tornado.py

```diff
@@ -96,6 +96,10 @@ def update_response(self, response, response_body=None):
         )
 
         if response_body:
+            if isinstance(response_body, list):
+                response_body = response_body[0]
```

Comment:
Is it possible that the passed response_body is the actual response that the user uses? 
Could be a good idea to use a different var and not **change** the response_body itself

Reply:
Also handle empty list use case